### DISCUSSION TITLE
Enable links to local files via drag and drop

### DISCUSSION THREAD

**Discussion by ddauber** [athensresearch/athens#837] [Feature Requests]
Create a link from Athens to any file on your local or external hard drive via drag and drop without creating a copy of this file within Athens (as is the case with images).

This comes in handy when working with unusual file formats or formats that are not ideally presented within Athens (e.g. a database of 3 million tweets with over 100 attributes per tweet).

**Comment by jsmorabito** [comment]
for reference here's some examples of how obsidian handles external files: 

- https://publish.obsidian.md/help/Plugins/Open+in+default+app
- https://publish.obsidian.md/help/Plugins/File+explorer
